packages/shared/consoleWithStackDev.js

```diff
@@ -8,11 +8,8 @@
 import ReactSharedInternals from 'shared/ReactSharedInternals';
 import {enableOwnerStacks} from 'shared/ReactFeatureFlags';
 
-let suppressWarning = false;
 export function setSuppressWarning(newSuppressWarning) {
-  if (__DEV__) {
-    suppressWarning = newSuppressWarning;
-  }
+  // TODO: Noop. Delete.
```

Comment:
Can we fully delete this?

Reply:
I see, a dependency on #30311

Reply:
I think we can fully delete it but someone needs to confirm in www/fb-rn